### DISCUSSION TITLE
Pass callback between javascript and go [v2]

### DISCUSSION THREAD

**Discussion by TechplexEngineer** [wailsapp/wails#1417] [Q&A]
Is it possible to pass a callback from javascript to go?

Here is the generated javascript method:
https://github.com/TechplexEngineer/BionicDash/blob/main/frontend/wailsjs/go/goNTCore/Client.d.ts#L45

Here is the corresponding golang function:
https://github.com/TechplexEngineer/goNTCore/blob/main/client.go#L565

And the type definition for `ListenerCallback`
 https://github.com/TechplexEngineer/goNTCore/blob/main/client.go#L67

Are the limitation of the javascript <-> go interface documented? (Sorry if I missed that)

**Comment by leaanthony** [comment]
Could you use events for this?
